### ISSUE TITLE
Feedback: "Open a modal dialog" (Modal Dialog Example, Test 1, V24.05.02) Remove Assertions for JAWS and NVDA

### ISSUE THREAD

**Issue by IsaDC** [w3c/aria-at#1063]
## Description of Behavior

The assertion "JAWS switched from virtual cursor active to PC cursor active" should be remove from PC mode.

## Test Setup

- Test File: [test-03-openModalDialog-jaws.collected.html](https://aria-at.netlify.app/tests/modal-dialog/test-03-openModalDialog-jaws.collected.html)
- AT: JAWS (Version 2024.2403.3)
- Browser: Chrome (Version 124)

**Comment by IsaDC** [comment]
Resolved by #1064
